src/requirements.txt

```diff
@@ -15,6 +15,8 @@ requests==2.18.4
 six==1.11.0
 toml==0.9.2
 urllib3==1.22
+# git+https://github.com/nrempel/von_agent#egg=von-agent
```

Comment:
Should remove this line.  We have von-agent added below, and this should not be propagated into permitify.